### PR TITLE
Login & signup

### PR DESCRIPTION
**Pull request by tushartiwari7**
Stylings for Login and Signup Pages

### CODE REVIEW

**Review by Bhakti199 — APPROVED** [review]
No issues found, This code looks fine to me.
Good to merge.

File: src/Components/Layout/AppLayout.css [hunk 1]
```diff
@@ -1,5 +1,5 @@
 .layout {
-  display: grid;
+  display: grid !important;
```
Comment (Manju-sree) [inline src/Components/Layout/AppLayout.css:2]: avoid using !important in css if possible.

File: src/Pages/Login/Login.tsx [hunk 2]
```diff
@@ -0,0 +1,62 @@
+import { Button, Checkbox, Form, Input, Typography } from "antd";
+import { FC } from "react";
```
Comment (Manju-sree) [inline src/Pages/Login/Login.tsx:2]: can provide proper naming here {FC}

**Review by Manju-sree — APPROVED** [review]
looks good to merge apart from minor changes mentioned in comment. please have a look into it.

File: src/Components/Layout/AppLayout.css (hunk 1 above)
Comment (tushartiwari7) [reply src/Components/Layout/AppLayout.css:2]: i have used `ant design` and they use inline css sometimes. to override that i did this! Though try to find some other way too.

File: src/Pages/Login/Login.tsx (hunk 2 above)
Comment (tushartiwari7) [reply src/Pages/Login/Login.tsx:2]: FC stands for Functional Component. React uses this naming.

**Pull request merged**